packages/php-wasm/universal/src/lib/load-php-runtime.ts

```diff
@@ -160,6 +161,9 @@ export async function loadPHPRuntime(
 	PHPRuntime.originalExit = PHPRuntime._exit;
 
 	PHPRuntime._exit = function (code: number) {
+		if (PHPRuntime?.webServer) {
```

Comment:
A more explicit name would make the intention clearer, e.g.:

```suggestion
		if (PHPRuntime.outboundNetworkProxyServer) {
```

Also, PHPRuntime shouldn't be undefined here so the question mark after PHPRuntime isn't needed.

Reply:
Updated! Thanks @adamziel